packages/domain-search/src/ui/domain-search-controls/filter-popover.tsx

```diff
@@ -0,0 +1,115 @@
+import {
+	Button,
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	CheckboxControl,
+	FormTokenField,
+} from '@wordpress/components';
+import { useI18n } from '@wordpress/react-i18n';
+import { FilterState } from '../../page/types';
+import { FilterPopoverLabel } from './filter-popover-label';
+import { FilterPopoverTld } from './filter-popover-tld';
+import type { TokenItem } from '@wordpress/components/build-types/form-token-field/types';
```

Comment:
Instead of importing the `TokenItem` type here and elsewhere, we can do the type assertion within the `handleTldsChange` callback so it always calls it with an array of strings.

Reply:
As we discussed, I've inlined that function and now the parameter type is inferred automatically.